app/components/avo/modal_component.rb

```diff
@@ -4,13 +4,7 @@ class Avo::ModalComponent < Avo::BaseComponent
   renders_one :heading
   renders_one :controls
 
-  attr_reader :width
-  attr_reader :body_class
-
-  def initialize(width: :md, body_class: nil)
-    @width = width
-    @body_class = body_class
-  end
```

Comment:
Why are we deleting `@body_class`?

Reply:
This was a mistake. I thought it wasn't being used anywhere until you pointed it out here. I have added this back.